src/components/video-embed/types.ts

```diff
@@ -0,0 +1,8 @@
+import { ReactPlayerProps } from 'react-player'
+
+export interface VideoEmbedProps extends ReactPlayerProps {
```

Comment:
I think it would be really valuable to add a link to the "Props" section of `ReactPlayer` README here, what do you think? Here's the link I'm suggesting: https://github.com/CookPete/react-player#props.

Reply:
Great call! 💖 ✅ Added